portal-api/src/v1/api.js

```diff
@@ -316,19 +316,19 @@ const getUtilisationReports = async (bankId, options) => {
   }
 };
 
-const getUtilisationReportById = async (_id) => {
+/**
+ * @param {number} id
+ * @returns {Promise<import('@ukef/dtfs2-common').UtilisationReportResponseBody>}
+ */
+const getUtilisationReportById = async (id) => {
   try {
-    if (!isValidMongoId(_id)) {
-      throw new Error(`Invalid MongoDB _id provided: '${_id}'`);
-    }
-
-    const response = await axios.get(`${DTFS_CENTRAL_API_URL}/v1/utilisation-reports/${_id}`, {
+    const response = await axios.get(`${DTFS_CENTRAL_API_URL}/v1/utilisation-reports/${id}`, {
       headers: headers.central,
     });
 
     return response.data;
   } catch (error) {
-    console.error(`Unable to get utilisation report with MongoDB _id '${_id}'`, error);
+    console.error(`Unable to get utilisation report with id '${id}'`, error);
```

Comment:
Unsure how much we care for this right now -- but if we throw this error up the chain, as appose to catching it and logging it here, further up the chain we can catch this AxiosError and assign it to the `cause` of an error up the chain. We can then keep applying this strategy as we go up to the controller level, where we can catch, log, and then handle these errors if needed to apply -- see `signInLinkController` I believe for a good example of this :) 

 For example, here, you could throw either an `ApiError` or `UtilisationReportNotFoundError` error depending on the response. We could then catch these errors at the controller level of this service, and if its a `UtilisationReportNotFoundError`, we can return a 404 (or bad req, or whatever status). If it's an `ApiError` we could let it default to a 500 by just rethrowing at the controller level.

Reply:
If this is a refactor thats wide in scope -- which it might be looking at the amount of files we've already have touched  -- please just leave this and resolve this comment :) Very happy not to worry about this right now

Reply:
The scope of this ticket is just to refactor the endpoint to hit the SQL db rather than mongodb and has already been quite a bit more work then estimated. I'm all for making improvements as we go, and your suggestions all seem sensible, but I also want to be wary of scope creep. @ChristianMcc Do you want me to review and refactor the error response handling for the flows or shall we leave it as is?

Reply:
Leave as is :)

Reply:
Leave as is for now please as it's something for 90% of functions in this file :) completely i agree i'd prefer to not catch errors in this file